server/src/main/java/org/opensearch/cluster/service/ClusterManagerThrottlingKey.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.cluster.service;
+
+/**
+ * Class to store the throttling key for the tasks of cluster manager
+ */
+public class ClusterManagerThrottlingKey {
+    private String taskThrottlingKey;
+    private boolean throttlingEnabled;
+
+    /**
+     * Class for throttling key of tasks
+     *
+     * @param taskThrottlingKey - throttling key for task
+     * @param throttlingEnabled - if throttling is enabled or not i.e. data node is performing retry over throttling exception or not.
+     */
+    public ClusterManagerThrottlingKey(String taskThrottlingKey, boolean throttlingEnabled) {
```

Comment:
The very idea of this class was to make this an inner class Throttler with only privately accessible constructor so that the only way to get an instance was through registration

Reply:
Made the constructor private, but we will need to expose the Default throttling key via static field from Throttler class as that will be needed for the tasks which are not on boarded on throttling framework. They will use this default key, and throttling will be bypassed for that default key.